src/test/slash-cmd.ts

```diff
@@ -0,0 +1,51 @@
+import { TOKEN } from './config.ts'
+
+export const CMD = {
+  name: 'blep',
+  description: 'Send a random adorable animal photo',
+  options: [
+    {
+      name: 'animal',
+      description: 'The type of animal',
+      type: 3,
+      required: true,
+      choices: [
+        {
+          name: 'Dog',
+          value: 'animal_dog'
+        },
+        {
+          name: 'Cat',
+          value: 'animal_dog'
+        },
+        {
+          name: 'Penguin',
+          value: 'animal_penguin'
+        }
+      ]
+    },
+    {
+      name: 'only_smol',
+      description: 'Whether to show only baby animals',
+      type: 5,
+      required: false
+    }
+  ]
+}
+
+// fetch('https://discord.com/api/v8/applications/783937840752099332/commands', {

```

Comment:
Change fetch(... to `Url: https://discord.com/api/v8/applications/783937840752099332/commands`

Reply:
That I was testing, yet to introduce a `Client#slash` for a rich Slash Commands management!

Reply:
Oops, okay